assets/js/lesson_3/calculator.js

```diff
@@ -0,0 +1,34 @@
+const userAction = prompt("Choose what you want to do: 'add', 'sub', 'mult', 'div'")
+
+if (userAction === 'add' || userAction === 'sub' || userAction === 'mult' || userAction === 'div') {
```

Comment:
This check is redundant because this is a duplicate. It will be better move it to else block, when all `if's` are false

Reply:
Actually I didn't understand this comment clearly. I wanted from user to enter one of those commands exactly, so i did check through ||, otherwise there'll be an error. If it's cheked, we will be on the next step, where we move calculations to variables. So if user entered 'sub', for instance, we'll excecute 'sub' operation and show 'alert' with result.
Hope i explain my idea correctly, this is some kind of extra training in technical English for me. If i didn't, I'll try it in Russian xD

Reply:
@andrew221b I'm going to speak more about if-else. I'll try to explain the idea